content/en/synthetics/private_locations/configuration.md

```diff
@@ -97,11 +97,20 @@ Discard SSL errors when private location is using a proxy to send requests to Da
 **Default**: `10`<br>
 Maximum number of tests executed in parallel.
 
+`enableStatusProbes`
+: **Type**: <br>
```

Comment:
Can we fill in the blank here? @peculiarism 

```suggestion
: **Type**: <br>
```

Reply:
Boolean 👍🏻